django/contrib/admin/options.py

```diff
@@ -250,9 +250,7 @@ def get_field_queryset(self, db, db_field, request):
         if related_admin is not None:
             ordering = related_admin.get_ordering(request)
             if ordering is not None and ordering != ():
-                return db_field.remote_field.model._default_manager.using(db).order_by(
-                    *ordering
-                )
+                return related_admin.get_queryset(request).order_by(*ordering)
```

Comment:
This change seems to totally disregard the 'db' parameter. So, it looks like it'll always use the default data source, which might lead to some surprises in projects with multiple databases when users want to query sources other than the default one.